src/App.js

```diff
@@ -99,35 +110,35 @@ const App = () => {
                 onClick={() => setToggle(!toggle)}
                 inputProps={{ "aria-label": "Toggle theme" }}
               />
-              <IconButton aria-label="account-management-button">
-                <AccountCircleIcon
-                  aria-label="account of current user"
-                  aria-controls="menu-account"
-                  aria-haspopup="menu"
-                  sx={{ color: "#fff" }}
-                />
-              </IconButton>
+              <a href="/profile">
+                <IconButton aria-label="account-management-button">
+                  <AccountCircleIcon
+                    aria-label="account of current user"
+                    aria-controls="menu-account"
+                    aria-haspopup="menu"
+                    sx={{ color: "#fff" }}
+                  />
+                </IconButton>
+              </a>
             </form>
           </nav>
         </header>
         <div className="Main-app-container">
-          <Router>
-            <Routes>
-              <Route exact path="/settings" element={<Settings />} />
-              <Route exact path="/profile" element={<UserProfile />} />
-              <Route exact path="/dashboard" element={<Dashboard />} />
-              <Route exact path="/terms" element={<NotFound />} />
-              <Route path="*" element={<SideNav />} />
-            </Routes>
-          </Router>
+          <SideNav />
           <main className="Main-app">
             <section className="Section-app">
-              <UserProfile toggle={toggle} />
-              <Dashboard toggle={toggle} />
-              <Friends toggle={toggle} />
-              <Profile toggle={toggle} />
-              <Settings toggle={toggle} />
-              <Post toggle={toggle} />
+              <Router>
+                <Routes>
+                  <Route exact path="/" element={<Dashboard />} />
+                  <Route exact path="/dashboard" element={<Dashboard />} />
+                  <Route exact path="/settings" element={<Settings />} />
+                  <Route exact path="/profile" element={<UserProfile />} />
+                  <Route exact path="/friends" element={<Friends />} />
+                  <Route exact path="/profile" element={<Profile />} />
+                  <Route exact path="/post" element={<Post />} />
+                  <Route exact path="/terms" element={<Terms />} />
+                </Routes>
```

Comment:
There is no 404 Not Found path/route?  I believe its <Route path="*"... <NotFound>>

Reply:
You are right, let me re-add this.

Reply:
It should pick up the NotFound component which should render something in the view saying basically oopsie we couldnt find your page//route, etc. etc. etc.  I believe this got commented out.